custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -1,14 +1,31 @@
 /* global moment, _ */
 
-function MonthModalController($location, $uibModalInstance, dateHelperService) {
+function MonthModalController($location, $uibModalInstance, dateHelperService, haveAccessToFeatures) {
     var vm = this;
 
     vm.months = [];
     vm.years = [];
     vm.monthsCopy = [];
     vm.showMessage = false;
+    var reportStartDates = {
+        'sdd': new Date(2019, 1),
+        'adolescent_girls': new Date(2019, 3),
+    };
+
     var isSDD =  $location.path().indexOf('service_delivery_dashboard') !== -1;
-    var startDate = $location.path().indexOf('service_delivery_dashboard') === -1 ? 2017 : 2019;
+    var isAdolescentGirls =  $location.path().indexOf('demographics/adolescent_girls') !== -1;
+
+
+    var startYear = 2017;
+
+
+    if (isSDD) {
+        startYear = reportStartDates['sdd'].getFullYear();
+    } else if (isAdolescentGirls && haveAccessToFeatures) {
```

Comment:
Hey, This is because we are changing the existing AG indicator on the dashboard and these new constraints are applied on the new one which is under QA at the moment that's why its under FF.

Reply:
Ah, makes sense thanks